### PR TITLE
Update release instructions

### PR DESCRIPTION
**Pull request by sgugger**
This updates the release instructions to the latest (just followed those for v0.22.0 this morning).

### CODE REVIEW

**Review by pacman100 — APPROVED** [review]
Thank you @sgugger for updating the release instructions, LGTM!

**Review by muellerzr — APPROVED** [review]
Thanks! As @BenjaminBossan I think we should definitely include a section on depreciation so it's easier to find them (as there's a few across the lib that haven't been addressed in a bit). I can make a PR today converting them all to TODO :)

**Comment by sgugger** [comment]
@BenjaminBossan The instructions are mostly for @LysandreJik who will do the releases from now on. It's not necessarily super comprehensive (for instance we only work from the main fork, so the comment related to that does not apply) and I mostly want to make sure he is okay with the comments before adding :-)

File: setup.py [hunk 1]
```diff
@@ -67,21 +67,27 @@
...
+#      rm -rf dist
+#      rm -rf build
 #      python setup.py bdist_wheel
 #      python setup.py sdist
-# 5. Upload the package to the pypi test server first:
-#      twine upload dist/* -r pypitest
-#      twine upload dist/* -r pypitest --repository-url=https://test.pypi.org/legacy/
-# 6. Check that you can install it in a virtualenv by running:
+# 6. Upload the package to the pypi test server first:
+#      twine upload dist/* -r testpypi
```
Comment (LysandreJik) [inline setup.py:84]: For `transformers` we're still using the `legacy` repository as we don't have a `pypitest` section in the `.pypirc`. 

It means generally using the command you erased:
```
#      twine upload dist/* -r pypitest --repository-url=https://test.pypi.org/legacy/
```

Have you managed to push to pypitest without a legacy URL and without a `.pypirc`?

File: setup.py [hunk 2]
```diff
@@ -67,21 +67,27 @@
...
+# 7. Check that you can install it in a virtualenv by running:
 #      pip install -i https://testpypi.python.org/pypi accelerate
```
Comment (LysandreJik) [inline setup.py:86]: For clean environments, may be worth putting:
```suggestion
#      pip install accelerate
#      pip uninstall accelerate
#      pip install -i https://testpypi.python.org/pypi accelerate
```

to ensure all deps can be installed first

**Review by LysandreJik — APPROVED** [review]
Looks good! Very similar to `transformers`. Thanks @sgugger.

File: setup.py (hunk 1 above)
Comment (LysandreJik) [reply setup.py:84]: Understood! I'll just try it with this one again next time and if it doesn't work I'll add the line that works for me once again.

**Review by BenjaminBossan — APPROVED** [review]
> The instructions are mostly for LysandreJik who will do the releases from now on. It's not necessarily super comprehensive (for instance we only work from the main fork, so the comment related to that does not apply) and I mostly want to make sure he is okay with the comments before adding :-)

Got it thx. LGTM

File: setup.py (hunk 2 above)
Comment (muellerzr) [reply setup.py:86]: I'd probably do:

> pip uninstall accelerate -y

So we can bypass the prompt? :)

**Pull request merged**